pytorch_lightning/utilities/imports.py

```diff
@@ -30,33 +31,36 @@ def _module_available(module_path: str) -> bool:
     """
     # todo: find a better way than try / except
     try:
-        mods = module_path.split('.')
-        assert mods, 'nothing given to test'
-        # it has to be tested as per partets
-        for i in range(len(mods)):
-            module_path = '.'.join(mods[:i + 1])
-            if importlib.util.find_spec(module_path) is None:
-                return False
-        return True
+        return find_spec(module_path) is not None
     except AttributeError:
+        # Python 3.6
+        return False
+    except ModuleNotFoundError:
+        # Python 3.7+
         return False
 
 
+def _get_version(package: str) -> LooseVersion:
+    return LooseVersion(pkg_resources.get_distribution(package).version)
```

Comment:
@Borda can you add a comment for why this is neccessary?

Reply:
just wrapper and make the following notations simpler